docs/dictionary.rst

```diff
@@ -0,0 +1,994 @@
+
+Dockstore Dictionary
+====================
+.. hlist:: 
+	:columns: 3
+
+	* :ref:`dict .dockstore.yml`
+	* :ref:`dict AGC`
+	* :ref:`dict Amazon Genomics CLI`
+	* :ref:`dict AnVIL Project`
+	* :ref:`dict API`
+	* :ref:`dict AWS`
+	* :ref:`dict BD Catalyst`
+	* :ref:`dict BDC`
+	* :ref:`dict BioData Catalyst`
+	* :ref:`dict Cancer Genomics Cloud`
+	* :ref:`dict categories`
+	* :ref:`dict CGC`
+	* :ref:`dict CLI`
+	* :ref:`dict cloud computing`
+	* :ref:`dict collection`
+	* :ref:`dict Common Workflow Language`
+	* :ref:`dict container`
+	* :ref:`dict Cromwell`
+	* :ref:`dict CWL`
+	* :ref:`dict DAG`
+	* :ref:`dict descriptor file`
+	* :ref:`dict Docker`
+	* :ref:`dict Docker container`
+	* :ref:`dict Docker image`
+	* :ref:`dict Dockerfile`
+	* :ref:`dict Dockstore CLI`
+	* :ref:`dict Dockstore GitHub App`
+	* :ref:`dict DOI`
+	* :ref:`dict DS-I Africa`
+	* :ref:`dict EC2`
+	* :ref:`dict egress`
+	* :ref:`dict eLwazi`
+	* :ref:`dict entry`
+	* :ref:`dict faceted search`
+	* :ref:`dict FAIR`
+	* :ref:`dict GA4GH`
+	* :ref:`dict Galaxy`
+	* :ref:`dict Galaxy workflow`
+	* :ref:`dict GCP`
+	* :ref:`dict Gen3`
+	* :ref:`dict GitHub App registration`
+	* :ref:`dict GitHub App tool`
+	* :ref:`dict GitHub App workflow`
+	* :ref:`dict immutable`
+	* :ref:`dict interoperable`
+	* :ref:`dict JSON`
+	* :ref:`dict Jupyter`
+	* :ref:`dict kernel`
+	* :ref:`dict Kids First`
+	* :ref:`dict labels`
+	* :ref:`dict launch with`
+	* :ref:`dict layer`
+	* :ref:`dict legacy registration`
+	* :ref:`dict legacy tool`
+	* :ref:`dict legacy workflow`
+	* :ref:`dict NCI`
+	* :ref:`dict NCPI`
+	* :ref:`dict Nextflow`
+	* :ref:`dict NFL`
+	* :ref:`dict NHGRI`
+	* :ref:`dict NHLBI`
+	* :ref:`dict NIH`
+	* :ref:`dict OICR`
+	* :ref:`dict ORCID`
+	* :ref:`dict organization`
+	* :ref:`dict parameter file`
+	* :ref:`dict preemptible`
+	* :ref:`dict Seven Bridges`
+	* :ref:`dict Spot Instance`
+	* :ref:`dict Task Execution Service`
+	* :ref:`dict Terra`
+	* :ref:`dict TES`
+	* :ref:`dict tool`
+	* :ref:`dict UCSC`
+	* :ref:`dict VM`
+	* :ref:`dict WDL`
+	* :ref:`dict WES`
+	* :ref:`dict workflow`
+	* :ref:`dict Workflow Description Language`
+	* :ref:`dict Workflow Execution Service`
+	* :ref:`dict YAML`
+
+.. _dict .dockstore.yml:
+
+.dockstore.yml
+--------------
+    This file is part of :ref:`dict GitHub App registration`. It indexes workflows or tools within a repository, including their optional test parameter files, and the author(s) of said workflows or tools.  
+
+Further reading: :doc:`/assets/templates/template </assets/templates/template>`  
+
+
+
+.. _dict AGC:
+
+AGC
+---
+*abbreviation for* :ref:`dict Amazon Genomics CLI`  
+
+
+
+
+.. _dict Amazon Genomics CLI:
+
+Amazon Genomics CLI
+-------------------
+    A :ref:`dict CLI`-based tool that supports launching bioinformatics-related workflows on :ref:`dict AWS` cloud infrastructure. Supports the :ref:`dict WES` standard, and can communicate with the :ref:`dict Dockstore CLI`.  
```

Comment:
Nit: The Dockstore CLI communicates with AGC. You could argue that communication is two-way, but it is the CLI that always initiates the communication/connection.

Reply:
To avoid getting too deep in the details of this, I think it's better to leave it as is unless there truly is no communication between the two.

Reply:
It doesn't communicate with the Dockstore CLI. You should drop it or reverse it.

Reply:
sorted this one out in Slack, marking as resolved